tools/setup/emoji/build_emoji

```diff
@@ -82,7 +82,12 @@ EMOJI_POS_INFO_TEMPLATE = """\
 }
 """
 
-# change directory
+# Make and enter emoji directory
```

Comment:
(this is very likely the solution to the cache problem encountered by @theopen-institute)